app/utils/versionChecker.js

```diff
@@ -1,18 +1,15 @@
 class VersionChecker {
-  latest (callback) {
```

Comment:
Ah, thanks, I am not that familiar with that stuff so I will try to learn from it. 

About **node**: I am copying what [electron](https://electron.atom.io/) version is using, so right now it's 7.4.0. Next version electron v1.7.0 beta will have 7.9.0.